gulpfile.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
 // init env variables
-require('dotenv').config();
```

Comment:
I don't believe I deleted this. I'm not sure why it's missing now. It was commented out with a todo to fix an issue with it.